frontend/src/components/layout/Layout.module.scss

```diff
@@ -152,6 +152,11 @@ $toastify-toast-close-button: var(--toast-close-button);
   // Thus, upping the z-index stack the header, and menus
   // expanding from it, on top again:
   z-index: 1;
+}
+
+.header-inner {
+  position: relative;
+  z-index: 2;
```

Comment:
suggestion (nit): Add a comment of the specific z-index value (See other z-index entries in the same file for context/pattern)